install.sh

```diff
@@ -1,11 +1,31 @@
 #!/bin/bash
 
-SSID="${1:-WiFiAPi}"
-PASSPHRASE="${2:-Raspberrypi}"
-IP_RANGE="${3:-192.168.1}"
+SSID="${1:-RaspberryPi}"
+PASSPHRASE="${2:-raspberry}"
+IP_RANGE="${3:-192.168.222}"
 INC="${4:-eth0}"
 WIFI="${5:-wlan0}"
 
+pause() {
+#	read -p "Appuyer sur une touche pour continuer ..." var
+	echo -n
+}
+
+begin() {
```

Comment:
Why did you extract this simple method as an additional function? Does not make it more readable.

Reply:
Yes and for maintenance. If you want to add or remove any this at each step of the script, you just have to change the begin and end function.